python/cudf_polars/cudf_polars/containers/dataframe.py

```diff
@@ -26,6 +26,20 @@
 __all__: list[str] = ["DataFrame"]
 
 
+def _create_polars_column_metadata(
+    name: str | None, dtype: pl.DataType | None
+) -> plc.interop.ColumnMetadata:
+    """Create ColumnMetadata preserving pl.Struct field names."""
+    if isinstance(dtype, pl.Struct):
+        children_meta = [
+            _create_polars_column_metadata(field.name, field.dtype)
```

Comment:
I think your implementation handles it, but can you add a test like `test_struct` verifying that nested structs are handled? Something like

```
t = pl.LazyFrame({"a": [{"x": {"i": 0, "j": 0}, "y": {"i": 0, "k": 1}}]}).collect()
t.schema
Schema([('a',
         Struct({'x': Struct({'i': Int64, 'j': Int64}), 'y': Struct({'i': Int64, 'k': Int64})}))])
```